samples/Nancy.Demo.Hosting.Kestrel/HomeModule.cs

```diff
@@ -8,7 +9,19 @@ public HomeModule()
         {
             Get("/", (args, ct) => Task.FromResult("Hello from Nancy running on CoreCLR"));
 
-            Get("/conneg/{name}", (args, token) => Task.FromResult(new Person() { Name = args.name }));
+            Get("/conneg/{name}", (args, token) => Task.FromResult(new Person() { Name = args.name });
+
+            Post("/", (parameters, token) =>

```

Comment:
Why are you using a Task-based route? Should just use a normal route and return `422` or `person` since it would resolve the `Post<object>(...)` overload